neptune-export/src/main/java/com/amazonaws/services/neptune/propertygraph/NodesClient.java

```diff
@@ -27,8 +27,7 @@
 import java.io.IOException;
 import java.util.*;
 
-import static org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__.select;
-import static org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__.valueMap;
+import static org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__.*;
```

Comment:
@iansrobinson Can you please update this to the specific imports required rather than the wildcard?